src/main/resources/static/test-scheme.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Article (
+  id INTEGER NOT NULL IDENTITY,
```

Comment:
I saw that you are using MySQL-driver in application.properties and I don't understand what mean `IDENTITY` in MySQL. I know that in MySQL use `AUTO_INCREMENT` for auto-filling like `IDENTITY` parameter in MS SQL. So I suppose you can change your `IDENTITY` column for `AUTO_INCREMENT` parameter and inserting new records without manually filling this column.

Reply:
@finr1r This SQL scheme used for HSQLDB Driver (needs for non-integration unit tests), it doesn't support AUTO_INCREMENT, only IDENTITY.